recipes/vsg/all/conandata.yml

```diff
@@ -8,3 +8,12 @@ sources:
   "1.0.5":
     url: "https://github.com/vsg-dev/VulkanSceneGraph/archive/refs/tags/v1.0.5.tar.gz"
     sha256: "ff58260fcb88d19d92c40a70bc40ff06abb1a8805568eb76862a036d13ada75b"
+  "1.0.6":
```

Comment:
These versions 1.0.6 and 1.0.7 are really required? Could you update only to 1.0.8? We kind of avoid updating to outdated versions, as most are unused.

Reply:
You're probably right. I'll remove them.